pyproject.toml

```diff
@@ -58,13 +58,13 @@ select = ["E", "F", "I"]
 combine-as-imports = true
 
 [tool.mypy]
-disallow_untyped_defs = true
+strict = true
 ignore_missing_imports = true
-show_error_codes = true
+python_version = "3.8"
 
 [[tool.mypy.overrides]]
 module = "starlette.testclient.*"
-no_implicit_optional = false
+implicit_optional = true
 
 [[tool.mypy.overrides]]
```

Comment:
So... Can we remove this?

Reply:
Actually, let's leave this here, and add a note on top of it mentioning the mypy issue, please.